cli/gobuster.go

```diff
@@ -177,6 +199,7 @@ func Gobuster(prevCtx context.Context, opts *libgobuster.Options, plugin libgobu
 	o := &outputType{
 		Mu:              outputMutex,
 		MaxCharsWritten: 0,
+		JSON:            opts.JSON,
```

Comment:
no need for that, you can access the property in the result worker from the gobuster variable.

Reply:
removed